src/Compilers/Core/AnalyzerDriver/AnalyzerExecutor.cs

```diff
@@ -495,8 +495,9 @@ internal static bool IsCanceled(Exception ex, CancellationToken cancellationToke
         internal static Diagnostic GetAnalyzerExceptionDiagnostic(DiagnosticAnalyzer analyzer, Exception e)
         {
             var descriptor = new DiagnosticDescriptor(AnalyzerExceptionDiagnosticId,
-                AnalyzerDriverResources.AnalyzerFailure,
-                AnalyzerDriverResources.AnalyzerThrows,
+                title: AnalyzerDriverResources.AnalyzerFailure,
+                messageFormat: AnalyzerDriverResources.AnalyzerThrows,
+                description: string.Format(AnalyzerDriverResources.AnalyzerThrowsDescription, e.ToString()),
```

Comment:
I think e.ToString() already includes some prefix, doesn't it (Includes type of exception and message)?

Also we don't seem to indicate the 'Type' of the exception in the primary (non-expanding) portion of the message. I think we should include this (i.e. Analyzer threw an exception of type '{0}' with message '{1}').